### PR TITLE
LPS-108070 New module for SameSite Cookies support

### PR DESCRIPTION
**Pull request by stian-sigvartsen**

### CODE REVIEW

**Comment by stian-sigvartsen** [comment]
ci:test

**Comment by csierra** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/portal/portal-samesite-cookies-support/src/main/java/com/liferay/portal/samesite/cookies/support/internal/configuration/SameSiteCookiesConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ */
+
+package com.liferay.portal.samesite.cookies.support.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Tomas Polesovsky
```
Comment (stian-sigvartsen) [inline modules/apps/portal/portal-samesite-cookies-support/src/main/java/com/liferay/portal/samesite/cookies/support/internal/configuration/SameSiteCookiesConfiguration.java]: Whops

Comment (csierra) [reply modules/apps/portal/portal-samesite-cookies-support/src/main/java/com/liferay/portal/samesite/cookies/support/internal/configuration/SameSiteCookiesConfiguration.java]: busted!

**Comment by csierra** [comment]
Superseded by https://github.com/csierra/liferay-portal/pull/622 ?

**Pull request closed without merging**